apps/web/src/pages/templates/components/EchoProjectWaitList.tsx

```diff
@@ -0,0 +1,125 @@
+import { Tooltip, Dropdown, CardTile } from '@novu/design-system';
+import { Badge } from '@mantine/core';
+
+import { TemplatesGroup, TemplateItem, GroupName } from './templates-store/templateStoreStyles';
+import { useSegment } from '../../../components/providers/SegmentProvider';
+import { useAuthContext } from '@novu/shared-web';
+
+const SEGMENT_EVENT = 'Button Clicked - [Echo promotion]';
+const segmentEventAction = {
+  dropDownItem: 'Workflows > Dropdown > Echo item',
+  modalItem: 'Workflows > Dropdown > All templates > Echo item',
+  cardTile: 'Workflows > No workflows yet > Echo item',
+};
+
+const handleEchoClick = (event: React.MouseEvent<HTMLButtonElement | HTMLDivElement>) => {
+  event.preventDefault();
+  window.open('https://novu.co/novu-echo-coming-soon?utm_campaign=echo_workflows', '_blank');
+};
+
+const NewBadge = (props) => {
+  return (
+    <>
+      <Badge
+        sx={{
+          padding: 5,
+          pointerEvents: 'none',
+          border: 'none',
+          background: '#dd2476',
+          color: 'white',
+          fontWeight: 'bold',
+          fontSize: '12px',
+          marginRight: props.marginRight ?? 10,
+        }}
+      >
+        NEW
+      </Badge>
+      Code Based
+    </>
+  );
+};
+
+const ToolTip = ({ children }) => {
+  return (
+    <Tooltip
+      label="Discover a new way that allows you to build powerful, highly customizable workflows on your local machine."
+      position="bottom"
```

Comment:
i would suggest showing it on the right and please split the text to multiple lines